src/test/java/util/xcmailr/util/SendRequest.java

```diff
@@ -70,26 +70,33 @@ public static void login(String email, String password)
         }
```

Comment:
We throw the exception in line 69 already, so there is no need to print the stack trace in line 68.